server/models/message.js

```diff
@@ -1,34 +1,37 @@
-const { UUID } = require('mongodb');
-const mongoose = require('mongoose');
-const { json } = require('stream/consumers');
+const { UUID } = require("mongodb");
+const mongoose = require("mongoose");
+const { json } = require("stream/consumers");
 
-const messageSchema = new mongoose.Schema({
-    sender: { 
-        type: String, 
-        ref: 'user'
+const messageSchema = new mongoose.Schema(
+  {
+    sender: {
+      type: String,
+      ref: "user",
     }, // maps to the username which sent the message
-    message_id: { 
-        type: UUID,
-        unique: true,
+    message_id: {
+      type: UUID,
```

Comment:
as before: we're storing our UUIDs as the `string` data type, not a UUID datatype